README.md

```diff
@@ -30,68 +30,68 @@ Otherwise, you may use cargo:
+This will require a password for encrypting the wallet. After the wallet is created you will be shown the mnemonic phrase.
+
+> Note: You will need your password for signing and account derivation, and you will need your mnemonic phrase if you wish to recover your wallet in the future.
+
+### Import a wallet
 
-You can also initialize a wallet with your existing mnemonic phrase by using `forc-wallet import`.
+To import a wallet from an existing mnemonic phrase, use:
+
+```sh
+forc-wallet import
+```
 
 > Note: `forc-wallet` adheres to the [Web3 Secret Storage Definition](https://ethereum.org/en/developers/docs/data-structures-and-encoding/web3-secret-storage) and accepts paths to wallet files that adhere to this standard.
 
 ### Create an account
 
-To create an account for the initialized wallet, you can run:
+To create an account for the wallet, you can run:
 
 ```sh
-forc-wallet new
+forc-wallet account new
 ```
 
-This will require your wallet password (the one that you choosed in the initialization step). This will always derive the next account.
-
-### Sign a transaction
+This will require your wallet password (the one that you chose during creation). This command will always derive the next account that has not yet been derived locally.
 
-To sign a transaction, you need to have the transaction ID. You can generate a transaction and get its ID using `forc-client`. Signing the transaction once you have the ID is simple:
+To list all accounts derived so far, use the following:
 
 ```sh
-forc-wallet sign --id <transaction_id> --account-index <account_index>
+forc-wallet accounts
 ```
 
-## Other useful commands
+> Note: When we "create" an account, we are really just *revealing* it. All accounts are derived deterministically based on the wallet's mnemonic phrase and derivation path. `forc-wallet` will cache the public addresses of derived accounts within `~/.fuel/wallets/accounts`.
 
-### List accounts
+### Sign a transaction
 
-To list all accounts derived so far:
+To sign a transaction, you must provide the transaction ID. You can generate a transaction and get its ID using `forc-client`. Signing the transaction once you have the ID is simple:
 
 ```sh
-forc-wallet list
+forc-wallet account <account_index> sign tx <transaction_id>
 ```
 
+## Other useful commands
+
 ### Get address of an account
 
-To retrieve the address of a specific account, you can use:
+To derive the address of a specific account, you can use:
 
 ```sh
-forc-wallet account --index <account_index>
+forc-wallet account <account_index>
 ```
 
 ### Get private key of an account
 
 To retrieve the private key of a specific account, you can use:
 
 ```sh
-forc-wallet export --account-index <account_index>
-```
-
-### Initialize from existing mnemonic phrase
-
-To initialize a new HD wallet from an existing mnemonic phrase:
-
-```sh
-forc-wallet import
+forc-wallet account <account_index> export-private-key
```

Comment:
Good point, I was carrying over the `export` terminology we previously had, but perhaps simply `forc-wallet account <index> private-key` is better here? E.g. we already use `forc wallet accounts` to print accounts, and `forc wallet account <ix>` to print an account at an index.

That said, unlike the `accounts` and `account <ix>` commands, we present the private key in a "discrete" temporary window so that it doesn't remain in the console output after it's shared.

Reply:
OK I've changed this from `export-private-key` to `private-key` for now!